docs/connecting-data/connecting-to-apis-and-external-data/servicenow-api.md

```diff
@@ -1,7 +1,150 @@
----
-description: How to Connect DataDistillr to the ServiceNow API
----
-
 # ServiceNow API
 
-Details coming soon!
\ No newline at end of file
+## Creating a ServiceNow account
+Set up an account with [ServiceNow](https://www.servicenow.com/).
+
+
+### Rate Limits
+To prevent excessive inbound REST API requests, set rules that limit the number of inbound REST API requests processed per hour. 
+You can create rules to limit requests for specific users, users with specific roles, or all users.
+
+## How to Connect DataDistillr to ServiceNow
+To set up a data source connection for ServiceNow, you will need to have:
+
+- A unique name for your data source connection to be used in queries
```

Comment:
Link 'unique name', 'instance name', 'username', and 'password' to sections below as done in the JIRA form for consistency